text/0000-core-builder.md

```diff
@@ -0,0 +1,46 @@
+# Ubuntu Core Builder
+
+## Summary
+
+Over the past, the Paketo project has created four different stacks and builders based on the Ubuntu Linux distribution. The difference is in the number of packages included which in turn is based on the requirements of the different buildpacks included.
+This RFC proposes to create a new stack repository that will create a single `build` image and a number of `run` images based on the Ubuntu distribution and replace the existing `static`, `tiny` and `base` stacks.
+This RFC further proposes to create a new builder repository that will create a single builder using the build image as a base layer and the `base` run image as the default `run` image.
+
+Users will be able to use the new builder to build and run their applications using any of the language family buildpacks included. Instead of requireing a decision for an optimised builder, they can "just" start and later decide on an optimised `run` image.
+
+## Motivation
+
+The Paketo project will be easier to use for users - they can start without being asked to decide on optimisation.
+The stacks and builders maintainer team will have less work to do - they will only need to maintain a single stack and builder instead of three.
+
+## Detailed Explanation
+
+We would replace the current `base`, `tiny` and `static` stacks with a single stack that includes all packages required by the buildpacks. Similarly, we would replace the current `base`, `tiny` and `static` builders with a single builder.
+
+## Rationale and Alternatives
+
+We could leave everything as it is. We would stick to maintaining four stacks repositories and eight builder repositories and creating new ones for every LTS we adopt. But that would increase the burden on the maintainers.
+
+## Implementation
+
+I would propose to create the new stack and builder as part of moving to Ubuntu Noble Numbat. I.e. 
+- we should create a repository `paketo-buildpacks/stack-ubuntu-noble`
```

Comment:
I think we should consider the implications of https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md with this change.

1. Does it make sense to continue using the word `stack` in our repo name. The repo name and image names would be tricky to change later without potentially breaking things.

2. While I think removing stacks should be a separate change for us, I think we should at least consider how that might go and if we can easily do it with this set up. The idea being to use this change to position us well for stack removal.

Reply:
Mmh, I was used to still using stack as the overarching term for the combination of build and run images... I briefly [asked on Slack](https://cloud-native.slack.com/archives/C033DV9EBDF/p1734385684806259) and [Heroku seems to use "Bae Images"](https://cloud-native.slack.com/archives/C033DV9EBDF/p1734389150511999?thread_ts=1734385684.806259&cid=C033DV9EBDF).

Following that, we would have this:
```suggestion
- we should create a repository `paketo-buildpacks/ubuntu-noble-base-images`
```

I have no strong opinion on the matter though. Maybe react with 👍 or 👎 to indicate if the above suggestion should be merged.

Reply:
@dmikusa Thoughts?

Reply:
> briefly [asked on Slack](https://cloud-native.slack.com/archives/C033DV9EBDF/p1734385684806259) and [Heroku seems to use "Bae Images"](https://cloud-native.slack.com/archives/C033DV9EBDF/p1734389150511999?thread_ts=1734385684.806259&cid=C033DV9EBDF).

👍  Yes, that's what I've heard used as well. Base image. I think they wanted to use more standard terminology. 

> I have no strong opinion on the matter though. Maybe react with 👍 or 👎 to indicate if the above suggestion should be merged.

👍  It's probably not a huge thing. It just seemed like a good opportunity.